WalletWasabi.Fluent/Views/Wallets/Home/History/CoinJoinDetailsView.axaml

```diff
@@ -69,7 +69,7 @@
       <c:PreviewItem Icon="{StaticResource btc_logo}"
                      Text="Mining fee paid"
```

Comment:
This is not the mining fee, this is the coinjoin fee (coordination fee + mining fee).
```suggestion
                     Text="Coinjoin fee:"
```